### PR TITLE
SIG Usability 2021 annual report

### PR DESCRIPTION
**Pull request by tashimi**
Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sig-usability/annual-report-2021.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+When was your last monthly community-wide update? (provide link to deck and/or recording)
+I do not recall 
+
+## Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+We need to finalize the PR on kubernetes/kubernetes to make Gaby formally a chair 
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We engage with membership via Slack and mailing list
```
Comment (parispittman) [inline sig-usability/annual-report-2021.md:32]: do you consider members of the group to be active participants on these channels? do work in subproject? how is that defined?

File: sig-usability/annual-report-2021.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+We need to finalize the PR on kubernetes/kubernetes to make Gaby formally a chair 
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We engage with membership via Slack and mailing list 
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+We are not focused on code PRs, we’re focused on staffing our user research study, which happens in Zoom. We’ve had steady growth of a small number of very senior user searchers from several different companies, and our goal is to do 2-4 in depth user research engagements a month.
```
Comment (parispittman) [inline sig-usability/annual-report-2021.md:35]: do you publish anywhere? can you link the ones that you did for 2020?

File: sig-usability/annual-report-2021.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+We’ve documented our user research procedures and invite people to sign up to do shadowing and help take notes. As they participate they are invited to graduate to leading interviews.
```
Comment (parispittman) [inline sig-usability/annual-report-2021.md:38]: can you share those documents here?

File: sig-usability/annual-report-2021.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+What programs do you participate in for new contributors?
+We are doing a Linux Foundation mentorship program, and new contributors are welcome at all of our meetings
```
Comment (parispittman) [inline sig-usability/annual-report-2021.md:41]: which program?

**Review by parispittman — CHANGES_REQUESTED** [review]

**Comment by dims** [comment]
@tashimi @hpandeycodeit can you please wrap up if possible today?

File: sig-usability/annual-report-2021.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Is your README accurate? have a CONTRIBUTING.md file?
+Yes, updated to reflect new meeting time
+https://github.com/kubernetes-sigs/sig-usability 
+
+All subprojects correctly mapped and listed in sigs.yaml?
+N/A
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+- Videos are all here: https://www.youtube.com/watch?v=cujQhg6JcgI&list=PL69nYSiGNLP0eY-U8DVJWHBwKvMDEtOxx 
+- We have small, interactive meetings, usually discuss and coordinate around our user research study that’s on-going
```
[the remark this answers is not in the document]
Comment (tashimi) [reply sig-usability/annual-report-2021.md:15]: No, we are presenting our data at the next Kubecon, and we will not be completely open sourcing our data in order to protect privacy of participants. We have future plans to scrub the data and open source it for others to use.

File: sig-usability/annual-report-2021.md (hunk 1 above)
Comment (tashimi) [reply sig-usability/annual-report-2021.md:32]: We don't have any formal sub-projects. Any one is welcome to participate. They can join the mailing list to get updates, or be in the slack channel, or attend meetings.

File: sig-usability/annual-report-2021.md (hunk 2 above)
Comment (tashimi) [reply sig-usability/annual-report-2021.md:35]: See above comment - we are not publishing data yet.

File: sig-usability/annual-report-2021.md (hunk 3 above)
Comment (tashimi) [reply sig-usability/annual-report-2021.md:38]: No, they are not open to the public. We take safe guarding the time and privacy of our research participants very seriously, and opening up our entire process could easily cause a troll problem.

File: sig-usability/annual-report-2021.md (hunk 4 above)
Comment (tashimi) [reply sig-usability/annual-report-2021.md:41]: CNCF LFX Mentoring Spring 2021

**Comment by tashimi** [comment]
@dims I believe all the open items have been addressed.

**Pull request merged**